packages/backend/src/tokens/request.ts

```diff
@@ -139,7 +139,8 @@ export async function authenticateRequest(
       const newUrl = new URL(authenticateContext.clerkUrl);
       newUrl.searchParams.delete(constants.QueryParameters.Handshake);
       newUrl.searchParams.delete(constants.QueryParameters.HandshakeHelp);
-      headers.append('Location', newUrl.toString());
+      headers.append(constants.Headers.Location, newUrl.toString());
+      headers.append(constants.Headers.CacheControl, 'no-store');
```

Comment:
@nikosdouvlis  Should we create a constant for the `no-store` value? Because it's used twice!